### ISSUE TITLE
Getting a 403 Forbidden when getting list of install apps.

### ISSUE THREAD

**Issue by matula** [microsoftgraph/microsoft-graph-docs#9460]
The graph token it authorized with 'User.ReadWrite.All', and I'm doing a GET request to 'https://graph.microsoft.com/beta/users/{userid}/teamwork/installedApps'. And it throws a 403.

`{
  "error": {
    "code": "UnknownError",
    "message": "",
    "innerError": {
      "date": "2020-08-11T19:28:08",
      "request-id": "ca6f85b6-9152-4330-a1c5-b6ce73378d30"
    }
  }
}`

The token works, since I can remove "teamwork/installedApps" and it returns the user info.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 27e0c7db-a498-3c5b-f414-c7edb352b71a
* Version Independent ID: 957eb16a-547f-b7f4-5871-c2d64e1446db
* Content: [List apps installed for user - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/user-list-teamsappinstallation?view=graph-rest-beta&tabs=http)
* Content Source: [api-reference/beta/api/user-list-teamsappinstallation.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/user-list-teamsappinstallation.md)
* Product: **microsoft-teams**
* Technology: **microsoft-graph**
* GitHub Login: @clearab
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by VaraPrasad-MSFT** [comment]
@matula ,
To get list of installed apps for the user you need to add all permissions as documented [here](https://docs.microsoft.com/en-us/graph/api/user-list-teamsappinstallation?view=graph-rest-beta&tabs=http#permissions). Could you please try adding the required permissions ?

**Comment by cho-minsoo** [comment]
The permissions  is normal, but I have the same error.

**Comment by JarbasHorst** [comment]
Hello @matula 

I have been able to reproduce the error from Graph Explorer using delegated permissions. Are you also using delegated permissions in your requests?  However, if I use application permissions, I am able to retrieve the installed apps for a user.

Regards
Jarbas Horst

**Comment by NiceJingle** [comment]
Our team is having the same issue, we have application permissions granted for this and it still returns the 403 unknown error. It's worked fine from January 2020 until this week.

**Comment by VaraPrasad-MSFT** [comment]
To get the list of apps installed for a user you need additional permissions 

For Delegate 
```
TeamsAppInstallation.ReadForUser, 
TeamsAppInstallation.ReadWriteForUser
```

also, could you please find below screen shot for your reference, and check the [docs ](https://docs.microsoft.com/en-us/graph/api/user-list-teamsappinstallation?view=graph-rest-beta&tabs=http#permissions)for more info.

![image](https://user-images.githubusercontent.com/67954779/90382849-f0576200-e09c-11ea-9068-e446982a0a2b.png)

**Comment by NiceJingle** [comment]
Adding those permissions did not fix the issue. In addition, the docs list to have one of those permissions, which we already had (user.read.all).

**Comment by chipallen** [comment]
> Adding those permissions did not fix the issue. In addition, the docs list to have one of those permissions, which we already had (user.read.all).

Hey @VaraPrasad-MSFT - Who can we escalate this to?  This API is clear broken and not working the way the document you provided.   We clearly have the User.Read.All (see @NiceJingle screen show) but it's still not working.

@eketo-msft @AckoMSFT @ArunMSFT @SeanMSFT @BenV-MSFT

**Comment by VaraPrasad-MSFT** [comment]
@chipallen ,The document is recently updated . [permissions](https://docs.microsoft.com/en-us/graph/api/user-list-teamsappinstallation?view=graph-rest-beta&tabs=http#permissions)  required to call this API. 


![image](https://user-images.githubusercontent.com/67954779/90540388-cab28180-e19e-11ea-94b5-a112663cbf6e.png)




I have not found mentioned permissions in your permissions list.

![image](https://user-images.githubusercontent.com/67954779/90540132-6263a000-e19e-11ea-9026-3305b8ace9de.png)



please add the required permissions as i mentioned above. Let me know if you face any issues.

**Comment by NiceJingle** [comment]
@VaraPrasad-MSFT 
I added those permissions as well
![Permissions2](https://user-images.githubusercontent.com/69685375/90554943-915d2000-e164-11ea-8998-55dfdebf8ffe.PNG)

After doing that I get the same error, that does not fix anything.
![stillerr](https://user-images.githubusercontent.com/69685375/90554970-9fab3c00-e164-11ea-9bd7-c2c8723a8cfe.PNG)

**Comment by Abhijit-MSFT** [comment]
@chipallen, @NiceJingle, @matula - Can any one of you just confirm if your token has the required permissions mentioned. 
This looks like bug. Your confirmation will help me processed creating bug.

**Comment by Gousia-MSFT** [comment]
Please re-open if this issue still persists

**Comment by scottsa-msft** [comment]
Please reopen this as we need to understand if this is a bug and the ETA for getting a fix.   @Abhijit-MSFT any updates?

**Comment by NiceJingle** [comment]
Hey have been having issues with my local and tunneling issues so testing has been slowed for me. On our dev I've added the permissions the the token and the call still fails. I ran it through the JWT to verify that the permissions are there which is posted below. We did switch the token since our old one was application level while these are delegated.
![tokenPermissionsJWT](https://user-images.githubusercontent.com/69685375/91587720-26aaa200-e925-11ea-9c80-984798a25688.PNG)
